backend/entities/dataset_files/csv_file.py

```diff
@@ -31,12 +31,10 @@ class CSVFile(DatasetFile):
     Attributes:
         df (pd.DataFrame): The DataFrame containing the dataset.
         categories (set): A set of protected categories present in the dataset.
-        bias_calculator (BiasCalculator): An instance of the BiasCalculator class.
 
     """
     df: pd.DataFrame
     categories: set
-    bias_calculator: BiasCalculator
```

Comment:
Makes sense since csv file just contains data and isn't related to bias calculator